packages/web-frontend/src/components/View/ChartWrapper/ChartWrapper.js

```diff
@@ -27,15 +27,24 @@ export class ChartWrapper extends PureComponent {
   getViewContent() {
     const { viewContent } = this.props;
     const { chartConfig, data } = viewContent;
+    const massagedData = this.sortData(this.removeNonNumericData(data));
     return chartConfig
       ? {
           ...viewContent,
-          data: this.sortData(data),
+          data: massagedData,
           chartConfig: parseChartConfig(viewContent),
         }
-      : { ...viewContent, data: this.sortData(data) };
+      : { ...viewContent, data: massagedData };
   }
 
+  removeNonNumericData = data =>
+    data.map(dataSeries =>
+      Object.entries(dataSeries).reduce((newDataSeries, [key, value]) => {
+        const isNonNumericData = isDataKey(key) && Number.isNaN(Number(value));
+        return isNonNumericData ? newDataSeries : { ...newDataSeries, [key]: value };
+      }, {}),
```

Comment:
I'm not sure how many entries are involved here, but we might want to use a `forEach` if there's the potential for there to be a lot (see https://beyondessential.slack.com/archives/CDGS6P56K/p1585867391005900)

Reply:
>  2000-3000 times faster!!

Wow! 

There are only as many entries here as legend entries (I have seen maybe 20 max), but will make the improvement anyway.